app/src/main/res/values/strings.xml

```diff
@@ -70,4 +70,5 @@ like to connect to:</string>
   <string name="add_service_data_field">Add Field</string>
   <string name="widget_config_service_error">A custom component is preventing service data from loading.</string>
   <string name="error_ssl">Unable to communicate with Home Assistant because of a SSL error.  Please ensure your certificate is valid.</string>
+    <string name="event_error">Unable to send event to HA.</string>
```

Comment:
Thinking about this again, the event really is an implementation detail of how we have designed actionable notifications. Should it be "Unable to notify HA of picked option." ?

Reply:
Done